src/documentation/dsls/sql/windowing-and-triggering.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: section
+title: "Beam DSLs: SQL"
+section_menu: section-menu/sdks.html
+permalink: /documentation/dsls/sql/windowing-and-triggering/
+---
+
+# Beam SQL: Windowing and triggering
+
+You can use Beam's windowing semantics in two ways:
+
+ - you can configure windowing on your input `PCollections` before passing them
+   to a `BeamSql` transform
+ - you can use windowing extensions in your windowing query, which will override
+   the windowing of your input `PCollections`
+
+Triggering can only be used by setting it on your input `PCollections`; there
+are no SQL extensions for specifying triggering.
+
+This section covers the use of SQL extensions to directly apply windowing.
+
+Beam SQL supports windowing functions specified in `GROUP BY` clause.
+`TIMESTAMP` field is required in this case. It is used as event timestamp for
+rows. 
+
+Supported windowing functions:
```

Comment:
This should have more explanation, a link to both Calcite and Beam docs about the windowing.